src/components/forms/AddPetForm.tsx

```diff
@@ -91,6 +89,10 @@ export const AddPetForm: React.FC<AddPetFormProps> = ({ onSuccess }) => {
             options={[
               { value: 'Cat', label: 'Cat' },
               { value: 'Dog', label: 'Dog' },
+              { value: 'Rabbit', label: 'Rabbit' },
+              { value: 'Hamster', label: 'Hamster' },
+              { value: 'Bird', label: 'Bird' },
+              { value: 'Guinea Pig', label: 'Guinea Pig' },
               { value: 'Other', label: 'Other...' },
             ]}
```

Comment:
🟡  I think that we will have to move options into supabase as well, or refactor it by pet.species so it always displays already existing pets. We can make default few options, 'dog', 'cat' then after 'other' is added also add it into the array.

Reply:
great suggestion! Let's keep this in mind for the future and do a similar thing for the list of breeds